src/pheval/cli_pheval_utils.py

```diff
@@ -18,10 +19,44 @@
     required=True,
     metavar="FILE",
     help="Path to the semantic similarity profile to be scrambled.",
+    type=Path,
+)
+@click.option(
+    "--output",
+    "-O",
```

Comment:
```suggestion
    "-o",
```

See below (other command), make sure not conflict